rholang/src/main/scala/coop/rchain/rholang/interpreter/Runtime.scala

```diff
@@ -369,13 +353,12 @@ object Runtime {
       extraSystemProcesses: Seq[SystemProcess.Definition[F]] = Seq.empty
   ) =
     for {
-      deployParametersRef <- Ref.of(DeployParameters.empty)
-      blockDataRef        <- Ref.of(BlockData.empty)
-      invalidBlocks       = InvalidBlocks.unsafe[F]()
-      urnMap              = basicProcesses ++ (stdSystemProcesses[F] ++ extraSystemProcesses).map(_.toUrnMap)
+      blockDataRef  <- Ref.of(BlockData.empty)
+      invalidBlocks = InvalidBlocks.unsafe[F]()
```

Comment:
If this is OK then simple `Ref` (instead of `Cell`)

Reply:
I don't follow? Where's the Cell?
Since this PR didn't change this code, and it doesn't seem critical, I think I'll skip this one :)

Reply:
Brain fart. Sorry, please ignore